src/DependencyInjection/Configuration.php

```diff
@@ -307,7 +306,6 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                         ->booleanNode('use_select2')->defaultTrue()->end()
                         ->booleanNode('use_icheck')->defaultTrue()->end()
-                        ->booleanNode('use_bootlint')->defaultFalse()->end()
```

Comment:
This might break the project, if you explicitly defined it before the update. We should keep it for now and show a deprecation warning.